src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -16,6 +16,7 @@ var ReactDOM = require('react-dom');
 var ReactDOMFeatureFlags = require('ReactDOMFeatureFlags');
 var ReactTestUtils = require('react-dom/test-utils');
 var PropTypes = require('prop-types');
+var JSDom = require('jsdom');
```

Comment:
jsdom is needed here to create a brand new document because by default jest-jsdom assumes every test is on the same document.